sdk/component/component.go

```diff
@@ -18,19 +18,19 @@ type (
 	// Reader allows reading vulnerability findings from a storage.
```

Comment:
can you shorten the commit message and add more context in the commit message please?